pandas/tests/io/json/test_normalize.py

```diff
@@ -378,6 +397,31 @@ def test_json_normalize_errors(self):
                       ['general', 'trade_version']],
                 errors='raise')
 
+    def test_missing_meta(self, address_missing_data):
+        # GH25468: If metadata is nullable with errors set to ignore, the null
+        # values should be numpy.nan values
+        result = json_normalize(
+            data=address_missing_data,
+            record_path='addresses',
+            meta='name',
+            errors='ignore')
+        ex_data = [
+            {'city': 'Massillon',
+             'number': 9562,
+             'state': 'OH',
+             'street': 'Morris St.',
+             'zip': 44646,
+             'name': 'Alice'},
+            {'city': 'Elizabethton',
+             'number': 8449,
+             'state': 'TN',
+             'street': 'Spring St.',
+             'zip': 37643,
+             'name': np.nan}
+        ]
+        expected = DataFrame(ex_data)
+        tm.assert_frame_equal(result, expected, check_like=True)
+
```

Comment:
Hmm...I see conflicting ways in which we have tested for correctness:

* `tm.assert_frame_equal`
* Converting to `dict` and then checking equality

I see you have chosen the former.  In which case, would it be possible to tweak the test so that you don't need to pass in `check_like=True` ?

Reply:
The reason I added `check_like=True` is because the test suit would fail on some platforms because the columns were not in the same order.

Perhaps I should convert to a `dict` instead?

Reply:
That also works.

Reply:
What is the `[columns]` needed here for? Would be preferable to just do `tm.assert_frame_equal(result, expected)`. 

Would be more explicit order-wise if you constructed the expected DataFrame from a list of lists rather than dict

Reply:
I will do that yes